Chart.yaml

```diff
@@ -19,7 +19,7 @@
 ---
 apiVersion: v2
 name: tika
-version: "latest-full"
+version: "1.25"
```

Comment:
@lewismc, OK I can change that, I just put 1.25 as a WAG because according to the helm spec (https://helm.sh/docs/topics/charts/#the-chartyaml-file) it needs a SemVer 2 version (https://semver.org/) so the `latest-full` value for `version` was preventing helm from deploying. I'm not all that familiar with helm so I'm not sure what the best way to parameterize that for automation.

Reply:
Yeah... neither am I if I am honest ...!